### ISSUE TITLE
Consider enabling checking of protobuf files by default

### ISSUE THREAD

**Issue by AlekSi** [Jason-Rev/vscode-spell-checker#236]
It works out-of-the-box for `.proto` files. Helped me notice a ton of typos in comments once I found spellchecker was disabled.

**Comment by Jason3S** [comment]
Personally, I would like to see the checker turned on for as many file types as possible.

At the same time, I try to consider general usage.

For the person editing the protobuf file, it makes a lot of sense to see spelling errors. But for the person including it in their project without the ability to edit, it could be annoying to see errors they have no control over.

**Comment by AlekSi** [comment]
That's true for any included/vendored files, and they can be in any language, aren't they? For example, checker is enabled by default for all Go files, and vendoring (putting dependencies in `vendor/` directory) is a recommended practice in Go.

**Comment by jskelcy** [comment]
@AlekSi were you able to enable this by adding `proto` to `cSpell.enableFiletypes`?

**Comment by jskelcy** [comment]
I figured it out I needed to enable for cSpell.enableFiletypes for `proto3` not just `proto`.
